Source/Scripts/cheat_core_items.lua

```diff
@@ -414,6 +414,67 @@ function cheat:cheat_damage_all_items(line)
   return false
 end
 
+-- Used by cheat_save_all_items and cheat_load_all_items
+_global_items_size = 1
+_global_items_classes = {}
+_global_items_amounts = {}
+
+-- ============================================================================
+-- cheat_save_all_items (To save all items you have at the end of 'a womans lot dlc')
+-- ============================================================================
+cheat:createCommand("cheat_save_all_items", "cheat:cheat_save_all_items()", nil,
+  "Dumps all items in inventory global variable that persists for the game session.",
```

Comment:
For simplicity and since most people don't know what a "global variable" is or what "dump" means in this context can you change the save/load documentation strings to read as:

"Saves inventory to temporary game memory. Intended for situations where the contents of your inventory will be lost due to game mechanics. i.e. A Woman's Lot (DLC). See cheat_load_all_items."

and 

"Loads inventory from temporary game memory. See cheat_save_all_items.".